src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$base-color: #060b35;
+$secondary-color: #616070;
+$button-buy-color: #00acdc;
+$button-buy-activecolor: #fff;
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Halvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 1px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__prduct-image {
+    width: 160px;
+    margin: 0 3px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $base-color;
+    text-align: left;
+    margin: 40px 0 0;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    width: 100%;
+    margin: 4px 0 0;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 100%;
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+
+    &__star:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin: 0 4px 0 0;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position-x: center;
+    background-position-y: center;
+  }
+
+  &__stars--1 :nth-of-type(-n + 1),
+  &__stars--2 :nth-of-type(-n + 2),
+  &__stars--3 :nth-of-type(-n + 3),
+  &__stars--4 :nth-of-type(-n + 4),
+  &__stars--5 :nth-of-type(-n + 5) {
```

Comment:
you can loop through with [@for](https://sass-lang.com/documentation/at-rules/control/for/)